### PR TITLE
ibm-ceph: ceph-mgr-callhome integration

### PR DESCRIPTION
**Pull request by guits**
This adds the required changes to include ceph-call-home in the IBM Ceph container image.

Fixes: #2147

### CODE REVIEW

**Comment by guits** [comment]
failure in arm64 job is unrelated and going to be addressed in a separate PR

File: ceph-releases/ALL/ubi9-ibm/daemon-base/__CEPH_IBM_PACKAGES__ [hunk 1]
```diff
@@ -0,0 +1 @@
+ceph-call-home
```
Comment (yaarith) [inline ceph-releases/ALL/ubi9-ibm/daemon-base/__CEPH_IBM_PACKAGES__:1]: @guits @ktdreyer is this the package name for `yum install`?
If so, I believe it should be `ceph-mgr-callhome`

Comment (ktdreyer) [reply ceph-releases/ALL/ubi9-ibm/daemon-base/__CEPH_IBM_PACKAGES__:1]: You're right, it's `ceph-mgr-callhome`
```suggestion
ceph-mgr-callhome
```

**Comment by ktdreyer** [comment]
We don't want this change in IBM Storage Ceph 6. The `BRANDING=ibm VERSION=6 ./contrib/compose-rhcs.sh` command shows it's affecting the `Dockerfile` for 6.

**Comment by yaarith** [comment]
> We don't want this change in IBM Storage Ceph 6. The `BRANDING=ibm VERSION=6 ./contrib/compose-rhcs.sh` command shows it's affecting the `Dockerfile` for 6.

@guits @ktdreyer not sure I understand how the changes to affect only IBM Storage Ceph 7 are implemented in this PR.

**Comment by yaarith** [comment]
the changes look good to me, @ktdreyer is the expert here, though :-)

**Comment by ktdreyer** [comment]
On this Git branch, I ran the following:

```
BRANDING=ibm VERSION=6 ./contrib/compose-rhcs.sh
BRANDING=ibm VERSION=7 ./contrib/compose-rhcs.sh
```

When I compare the `staging` contents to what is in dist-git today, the two diffs looks correct. The callhome additions are present for reef and absent for quincy.

**Review by ktdreyer — APPROVED** [review]

**Pull request merged**